scripts/release/release_info.py

```diff
@@ -1,90 +1,205 @@
+        UPGRADE_HOOK_IMAGE,
+        upgradehook_build_info.repositories[0],
+        upgradehook_build_info.platforms,
+        latest_upgrade_hook_version(release_data),
+    )
+
+    # add readiness image info
+    readiness_build_info = build_info.images[READINESS_PROBE_IMAGE]
+    add_image_info(
+        release_info_output,
+        READINESS_PROBE_IMAGE,
+        readiness_build_info.repositories[0],
+        readiness_build_info.platforms,
+        latest_readiness_version(release_data),
+    )
+
+    # add search image info
+    add_image_info(
+        release_info_output,
+        SEARCH_IMAGE,
+        SEARCH_IMAGE_REPOSITORY,
+        ["linux/arm64", "linux/amd64"],
+        latest_search_version(release_data),
+    )
+
+    add_image_info(
+        release_info_output,
+        MONGODB_ENTERPRISE_SERVER_IMAGE,
+        MONGODB_ENTERPRISE_SERVER_REPOSITORY,
+        ["linux/arm64", "linux/amd64"],
+        latest_enterprise_server_version(release_data),
+    )
+
+    release_info_output = add_om_agent_mappings(release_data, release_info_output)
+
+    return release_info_output
+
+
+def add_image_info(release_info_output, name, repository: str, platforms, version):
+    digest = manifest_list_digest_for_image(f"{repository}:{version}")
+    release_info_output["images"][name] = {
+        "repoURL": repository,
+        "platforms": platforms,
+        "tag": version,
+        "digest": digest,
+    }
+
+
+def add_om_agent_mappings(release_data, output):
+    om_agent_mapping = release_data["latestOpsManagerAgentMapping"]
+    output["latestOpsManagerAgentMapping"] = om_agent_mapping
 
     return output
 
 
+def latest_enterprise_server_version(release_data):
+    return release_data["supportedImages"]["mongodb-enterprise-server"]["versions"][-1]
+
+
+def latest_readiness_version(release_data):
+    return release_data["readinessProbeVersion"]
+
+
+def latest_upgrade_hook_version(relese_data):
+    return relese_data["versionUpgradeHookVersion"]
+
+
+def latest_om_version(release_data):
+    return release_data["supportedImages"]["ops-manager"]["versions"][-1]
+
+
+def latest_agent_version(release_data):
+    newest_om_version = release_data["supportedImages"]["ops-manager"]["versions"][-1]
+    newest_om_mapping = release_data["supportedImages"]["mongodb-agent"]["opsManagerMapping"]["ops_manager"][
+        newest_om_version
+    ]
+    return newest_om_mapping["agent_version"]
+
+
+def latest_search_version(release_data):
+    return release_data["search"]["version"]
+
+
+# manifest_list_digest_for_image returns manifest list digest for the passed image. Returns
+# empty string if there was an error figuring that out.
+def manifest_list_digest_for_image(image: str) -> str:
+    builder = DockerImageBuilder()
+    try:
+        digest = builder.get_manfiest_list_digest(image)
+    except Exception as e:
+        logger.error(f"There was an error, figuring out manifest list digest for image {image}. Error: {e}")
```

Comment:
q: so with this implementation we are fine to produce `release-info.json` that is missing some of the digests for images?

Reply:
For this release I think it's ok to not mention the digest for the image versions that are mentioned under OMAgentMapping. We can add that in the next release if required. Is that what you are talking about? I don't think we are missing digest in any other image.

Reply:
ohh, I understand you now. That's what my plan was, that if we are not able to find out the digest for an image we can have empty digest in release-info.json. Do you think we should fail instead?

Reply:
I don't know either. Can this actually happen with our current images? 

I would ask @vinilage and @mircea-cosbuc for guidance